core/primitives/res/runtime_configs/parameters.txt

```diff
@@ -119,6 +119,7 @@ wasm_storage_iter_next_base: 0
 wasm_storage_iter_next_key_byte: 0
 wasm_storage_iter_next_value_byte: 0
 wasm_touching_trie_node: 16_101_955_926
+wasm_read_cached_trie_node: 2_280_000_000
```

Comment:
Why did this move?

Reply:
See https://github.com/near/nearcore/pull/8115